### ISSUE TITLE
devon-ide broken in cygwin

### ISSUE THREAD

**Issue by hohwille** [devonfw/devon-ide#197]
```
Creating main workspace directory
devon-ide has environment variables have been set for /cygdrive/c/projects/cool-project

hohwille@laptop /cygdrive/c/projects/cool-project/workspaces/main
$ devon eclipse
ERROR: DEVON_IDE_HOME does not exist!
/cygdrive/c/projects/beta5
It seems as if you original installation has been moved.
Unknown command eclipse

hohwille@laptop /cygdrive/c/projects/cool-project/workspaces/main
$ devon
Creating main workspace directory
devon-ide has environment variables have been set for /cygdrive/c/projects/cool-project

hohwille@laptop /cygdrive/c/projects/cool-project/workspaces/main
$ devon help
Unknown command help

hohwille@laptop /cygdrive/c/projects/cool-project/workspaces/main
$ devon ide
Unknown command ide
```

**Comment by hohwille** [comment]
Fixed with PR #201

**Comment by hohwille** [comment]
See also #199
